src/gql/apollo.js

```diff
@@ -13,17 +13,15 @@ Vue.use(VueApollo)
 const graphqlHost = urlParams => urlParams.graphql || config.graphqlHost
 
 const makeHttpLink = urlParams => {
-  let prefix = config.development ? `http` : `https`
   const host = graphqlHost(urlParams)
-  const uri = `${prefix}://${host}`
+  const uri = host
```

Comment:
@colw @mariopino for awareness: `?graphql` and `VUE_APP_GRAPHQL_URL` now need a `http://` or `https://` in front